### PR TITLE
Clarify file extensions that can be crawled in SPO

### PR DESCRIPTION
**Pull request by TroyStarr**
Clarify that .eml and .mht file types can be crawled and parsed in SPO. Also some general formatting improvements in the article.

### CODE REVIEW

File: SharePoint/SharePointServer/technical-reference/default-crawled-file-name-extensions-and-parsed-file-types.md [hunk 1]
```diff
@@ -13,158 +13,132 @@ ms.prod: sharepoint-server-itpro
 ms.localizationpriority: medium
 ms.collection: IT_Sharepoint_Server_Top
 ms.assetid: 19d553a7-7001-4b07-a03d-616b865b05ae
-description: "Learn which file name extensions SharePoint Server crawls by default and which file types it parses by default."
+description: "Learn which file name extensions SharePoint Server and SharePoint in Microsoft 365 crawls by default and which file types it parses by default."
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/technical-reference/default-crawled-file-name-extensions-and-parsed-file-types.md:16]:
```suggestion
description: "Learn which file name extensions SharePoint Server and SharePoint in Microsoft 365 crawl by default and which file types it parses by default."
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @TroyStarr, kindly commit the suggested change. Many thanks.

**Comment by TroyStarr** [comment]
Thanks for the copy review @JohanFreelancer9.  I've committed the change you suggested.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**